src/tests/Microsoft.Agents.BotBuilder.Tests/TurnContextTests.cs

```diff
@@ -529,6 +530,93 @@ public async Task SendActivityAsync_ShouldThrowExceptionInOnSend()
             await Assert.ThrowsAsync<Exception>(() => context.SendActivityAsync(TestMessage.Message()));
         }
 
+        [Fact]
+        public async Task SendActivityAsync_ShouldCreateActivityWithParameters()
+        {
+            var adapter = new SimpleAdapter();
+            var context = new TurnContext(adapter, new Activity());
+            Assert.False(context.Responded);
+            var textMessage = "test activity created with parameters";
+            await context.SendActivityAsync(textMessage, textMessage, InputHints.AcceptingInput);
+
+            Assert.True(context.Responded);
+        }
+
+        [Fact]
+        public async Task SendActivitiesAsync_ShouldThrowExceptionOnEmptyActivitiesList()
+        {
+            var adapter = new SimpleAdapter();
+            var context = new TurnContext(adapter, new Activity());
+           
+            await Assert.ThrowsAsync<ArgumentException>(async () => await context.SendActivitiesAsync([]));
+        }
+
+        [Fact]
+        public async Task SendActivitiesAsync_ShouldSendActivitiesWithExpectRepliesMode()
+        {
+            var adapter = new SimpleAdapter();
+            var context = new TurnContext(adapter, new Activity() { DeliveryMode = DeliveryModes.ExpectReplies });
+            var message = new Activity
+            {
+                Type = ActivityTypes.InvokeResponse,
+                Id = "expect replies message",
+                Text = "test",
+            };
+
+            var response = await context.SendActivitiesAsync([message]);
```

Comment:
When there's feedback for one test, please if it applies to the rest. In this case, the response variable is never used and could be omitted. This was marked for a previous test.